Lvl99GameJam/Assets/Scripts/UI Scripts/GridSpaceInteractions.cs

```diff
@@ -81,6 +91,28 @@ private void DestroyPlacementIfNecessary()
         }
     }
 
+    private void ShowSpritePreviewIfNecessary()
+    {
+        if (!dummyPlacement)
```

Comment:
We don't want to show preview if there's already a placement in this grid square, right? Perhaps check `placement`?